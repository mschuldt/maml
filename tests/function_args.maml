arduino = Arduino(desktop=True)

@function
def ftest(a,b):
    print_i(a)
    print_i(b)
    return a+b

@block(once)
def test():
    print_s("calling ftest:")
    x = 3
    n = ftest(1, x)
    print_s("returned:")
    print_i(n)
    die(0)

    
arduino.send(ftest)
arduino.send(test)
