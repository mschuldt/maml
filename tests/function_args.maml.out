Initializing avm...
Ready.

calling ftest:
1
3
returned:
4
bye.
